packages/jaeger-ui/config-overrides-antd-vars.less

```diff
@@ -22,3 +22,7 @@
 
 // Table
 @table-row-hover-bg:#e5f2f2;
+
+// Font
+// see https://github.com/ant-design/antd-init/tree/master/examples/local-iconfont
+@icon-url: "/iconfont/iconfont"
```

Comment:
wouldn't this break people who're using `--query.base-path` option?

Reply:
I think it will. But I don't known enough about this antd library to known how to make that configurable. I think its worse to use the cdn than working around that at the http server level.